source/blog/2018-05-08-release-notes.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: May 2018 Release Notes - Version 18.5.0
+published: true
+post_author:
+  display_name: Vladimir Kostyukov
+  twitter: vkostyukov
+tags: Releases, Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+Good news, everyone! Finagle & Finatra May releases are here!
+
+As usual, all the corresponding Github projects for Finagle, Finatra, Util, Twitter Server, and
+Scrooge have their release pages updated:
+
+ * [Finagle 18.5.0][finagle]
+ * [Util 18.4.0][util]
```

Comment:
This scared me for a minute—I thought we were back to mismatched versions! But it's just a copy-paste error, I believe.